.github/workflows/initcontainer.yml

```diff
@@ -0,0 +1,131 @@
+
+      - name: Resolve latest contracts
+        run: |
+          npm install --save-exact \
+            @keep-network/keep-core@${{ steps.upstream-builds-query.outputs.keep-core-contracts-version }} \
+            @keep-network/keep-ecdsa@${{ steps.upstream-builds-query.outputs.keep-ecdsa-contracts-version }} \
+            @keep-network/tbtc@${{ steps.upstream-builds-query.outputs.tbtc-contracts-version }}
+
+      - name: Copy selected contracts
+        run: |
+          cp ./node_modules/@keep-network/keep-core/artifacts/KeepToken.json .
+          cp ./node_modules/@keep-network/keep-core/artifacts/TokenStaking.json .
+          cp ./node_modules/@keep-network/keep-ecdsa/artifacts/BondedECDSAKeepFactory.json .
+          cp ./node_modules/@keep-network/keep-ecdsa/artifacts/KeepBonding.json .
+          cp ./node_modules/@keep-network/tbtc/artifacts/TBTCSystem.json .
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Login to Google Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.GCR_REGISTRY_URL }}
+          username: _json_key
+          password: ${{ secrets.KEEP_TEST_GCR_JSON_KEY }}
+
+      - name: Build and publish initcontainer
+        uses: docker/build-push-action@v2
+        env:
+          IMAGE_NAME: initcontainer-provision-keep-ecdsa-${{ env.CHAIN }}
+        with:
+          context: ./infrastructure/kube/templates/keep-ecdsa/initcontainer/provision-keep-ecdsa/
+          # GCR image should be named according to following convention:
+          # HOSTNAME/PROJECT-ID/IMAGE:TAG
+          # We don't use TAG yet, will be added at later stages of work on RFC-18.
+          tags: ${{ env.GCR_REGISTRY_URL }}/${{ env.GOOGLE_PROJECT_ID }}/${{ env.IMAGE_NAME }}
+          labels: |
+            revision=${{ github.sha }}
+          push: true
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new
+
+      - name: Notify CI about completion of the workflow
+        uses: keep-network/ci/actions/notify-workflow-completed@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
+        with:
+          module: "github.com/keep-network/keep-ecdsa-initcontainer"
+          url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+          environment: ${{ github.event.inputs.environment }}
+          upstream_builds: ${{ github.event.inputs.upstream_builds }}
+          upstream_ref: ${{ github.event.inputs.upstream_ref }}
+          version: ${{ needs.contracts-migrate-and-publish.outputs.version }}
```

Comment:
`contracts-migrate-and-publish.outputs.version` was removed from the contracts workflow. Maybe we could use `steps.upstream-builds-query.outputs.keep-ecdsa-contracts-version` instead?

Reply:
Good catch and the suggestion. Fixed in 2848b682da61d45d64a1bd9ba045d3cd7b3ddf80.